biogrid/apps/biogrid-mvp/src/app/pages/input-page/input-page.css

```diff
@@ -0,0 +1,49 @@
+.input-page {
+  background-color: #f2f2f2;
+  border-radius: 5px;
+  font-family: sans-serif;
+  margin: auto;
+  padding: 20px;
+  text-align: center;
+  width: 900px;
+}
+
+body {
+  background-color: #3b6d6d;
+}
+
+br {
+  clear: both;
+}
+
+label {
+  display: block;
+  text-align: left;
+}
+
+.submitButton {
+  text-align: left;
+}
+
+.startDatePicker {
+  float: left;
```

Comment:
Nit: A good rule of thumb is to never use floats btw.  They have weird ripple effects

Reply:
In addition to what Casey said, look into css grid layout to replace the floats

Reply:
Will do thanks